cmd/nerdctl/volume_inspect.go

```diff
@@ -40,23 +40,24 @@ func newVolumeInspectCommand() *cobra.Command {
 	return volumeInspectCommand
 }
 
-func processVolumeInspectCommandOptions(cmd *cobra.Command) (types.VolumeInspectCommandOptions, error) {
+func processVolumeInspectCommandOptions(cmd *cobra.Command) (types.VolumeInspectOptions, error) {
```

Comment:
Do we also need to rename these `process${Command}CommandOptions` to `process${Command}Options`?